tests/Google/Ads/GoogleAds/Util/V1/GoogleAdsErrorsTest.php

```diff
@@ -105,4 +103,28 @@ public function testFromFailureWithInvalidOperationIndex()
         $errors = GoogleAdsErrors::fromFailure(1, $this->failure);
         $this->assertCount(0, $errors);
     }
+
+    public function testFromFailureWithMutateOperation()
+    {
+        $failureWithMutateOperation = new GoogleAdsFailure([
```

Comment:
Only thing that changes compared to the existing "failure" instance is the "field_name" value? It might be better factorizing them to avoid code duplication.

Reply:
Right, done.